roles/vmrole/templates/vm-template-windows2012r2.yaml.j2

```diff
@@ -44,12 +44,12 @@ objects:
                 tickPolicy: catchup
               hyperv: {}
           cpu:
-            cores: ${{CPU_CORES}}
+            cores: ${CPU_CORES}
```

Comment:
When processing via `oc process`, this will not work - the value will be escaped by `"` (quotes). Numeric value is expected.